src/scripts/notify.sh

```diff
@@ -3,6 +3,25 @@ LOG_PATH=$(mktemp -d)
 JQ_PATH=/usr/local/bin/jq
 POST_TO_SLACK_LOG=post-to-slack.json
 
+stdmsg() {
```

Comment:
`echo` should be avoided in shell scripting because its behavior is inconsistent across implementations; see [this Stack Exchange question](https://unix.stackexchange.com/questions/65803/why-is-printf-better-than-echo) for more details.